ansible_community_meetings.ics

```diff
@@ -8,11 +8,11 @@ DURATION:PT1H
 RRULE:FREQ=WEEKLY
 DESCRIPTION:Project:  Ansible Security Automation working group\nChair:  A
  dam Miller (maxamillion)\, Sumit Jaiswal (justjais)\, Abhijeet Kasurde (ak
- asurde)\nDescription:  \nWe discuss everything about Information Security 
+ asurde)\nDescription:  \nWe discuss everything about Information Security
```

Comment:
I wonder whether these trailing spaces are important and need to stay. Looking at the other lines which don't have spaces, and where words are just broken by a line end, I think they need to stay.

Reply:
I think you're right.  Maybe adding spaces to the beginning of the lines will be easier to maintain as we will know they're intentional (and IDEs are less likely to remove those automatically).

Reply:
Oof, my editor must have "done me a favor" there - I didn't intend to change the whitespace. Let me fix that.

Reply:
https://icalendar.org/iCalendar-RFC-5545/3-1-content-lines.html

Reply:
@abadger I thought that this file was auto-generated anyway from the YAML file, so usually nobody should edit it manually.

Reply:
Yep, you are correct.